db/migrate/20201109093337_create_trucks.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTrucks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :trucks do |t|
+      t.string :license_plate
+      t.integer :capacity
+      t.references :model_type
+      t.string :color
+      t.string :service_date
+      t.integer :status
+      t.references :drivers
+      t.timestamps null: false
```

Comment:
when you add to the migration you need to remember to re-run the migration as the schema has not been updated
To rerun you can either:
`rails db:drop && db:create`
or
`rails db:rollback` and then `rails db:migrate` again

Reply:
am i supposed to have 2 migrate files after migrating again?
because i did the roll back and migrate but the newer file didnt show up

Reply:
nah it should update the `schema.rb` with the updated table